### PR TITLE
Adding new VirtualMachines.POST.SimulateEviction action to compute api-version: 2019-12-01

### PR DESCRIPTION
**Pull request by sandeepkv93**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
LGTM, wait for ARM feedback. Meanwhile, @sandeepkv93 could you pls fix CI?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 1]
```diff
@@ -3439,6 +3439,48 @@
...
+            "description": "The name of the virtual machine."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```
Comment (KrisBash) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json:3471]: Why use a  `no content` response?  How does the user know the result of the request?

**Review by KrisBash — CHANGES_REQUESTED** [review]
Please see comment

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json (hunk 1 above)
Comment (sandeepkv93) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json:3471]: @KrisBash, this is the API to Simulate the Eviction in same way as how Spot Virtual Machine is evicted due to capacity or price reasons. The success from this API means, the request is sent successfully to the backend, and once eviction is triggered, customer can get a 30 seconds in-vm notification via Azure metadata service. Basically, this API's success response basically mean that that request is received successfully and no other payload is needed in the response.

And, HTTP Status 204 (No Content) indicates that the server has successfully fulfilled the request and that there is no content to send in the response payload body. Therefore it makes more sense return HTTP Status 200 (OK)

**Review by hyonholee — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]
LGTM

**Pull request merged**